README.md

```diff
@@ -0,0 +1,46 @@
+# mammosunits
+
+`mammosunits` is a units package built on top of the powerful Astropy units.
+It provides additional functionality for magnetic units.
+
+## Clone and Install 
+#### 1. Clone the repository
+
+To get started clone the `mammosunits` repository via `ssh`:
```

Comment:
Note for using `ssh`: this will only work for the people who have the right to push to the repo directly. For general users this will have to be `https`.